hash/hash_blake2b.c

```diff
@@ -0,0 +1,343 @@
+/**
+ * @addtogroup hash Hash algorithms
+ * @{
+ * @file hash_blake2b.c
+ * @author [Daniel Murrow](https://github.com/dsmurrow)
+ * @brief [Blake2b cryptographic hash
+ * function](https://www.rfc-editor.org/rfc/rfc7693)
+ */
+#include <assert.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert.h>    /// for assert
```